site/tsconfig.json

```diff
@@ -1,24 +1,24 @@
 {
   "compilerOptions": {
     "target": "ES2020",
-    "useDefineForClassFields": true,
     "lib": [
       "DOM",
       "DOM.Iterable",
       "esnext"
     ],
-    "module": "esnext",
-    "skipLibCheck": true,
-    "moduleResolution": "bundler",
     "allowImportingTsExtensions": true,
-    "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": true,
     "jsx": "react-jsx",
-    "strict": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
```

Comment:
By changing these to
```
    "module": "NodeNext",
    "moduleResolution": "NodeNext",
```
it did the trick.

cc @kyhyco @cnasc

Reply:
ahhh nice one!